src/test/java/SetTest.java

```diff
@@ -0,0 +1,77 @@
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set은 숫자를 잘 반활할까?")
+    void case1() {
+        //given
+
+        //when
+        int size = numbers.size();
+
+        //then
+        Assertions.assertThat(size).isInstanceOf(Integer.class);
```

Comment:
해당 테스트의 의미인 `Set은 숫자를 잘 반환할까?`의 의미로만 접근한다면,
```suggestion
       //when
        Optional<Integer> member = numbers.stream().findAny();

        // then
        assertThat(member.isPresent()).isTrue();
        Assertions.assertThat(member.get()).isInstanceOf(Integer.class);
```
의 구조처럼 존재하는 지 확인 + 확인 후 get했을 때 같은 타입인지 확인하는 건 어떨까요?